Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -30,15 +30,23 @@ - (NSArray *)managedObjectContext:(NSManagedObjectContext *)context
                                       description:[self descriptionFromRequest:request]];
     }];
 
+    NSArray *result = original(request, error);
```

Comment:
The problem with this new solution is that you logging the beginning of the write process after it occurs. 
Both `started` and `finished` logs will occur basically at the same time.

Reply:
Good point